src/main/resources/changesets/changelog-consent-61.0.xml

```diff
@@ -0,0 +1,56 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="61.0" author="raejohanek">
+        <validCheckSum>ANY</validCheckSum>
+        <createTable tableName="institution">
+            <column name="institution_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="institution_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="it_director_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="it_director_email" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="create_user" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="create_date" type="DATETIME">
+                <constraints nullable="false"/>
+            </column>
+            <column name="update_date" type="DATETIME"/>
+            <column name="update_user" type="bigint"/>
+        </createTable>
```

Comment:
Good practice here is to add a rollback. In our case, we'd drop the table along with all constraints. https://docs.liquibase.com/commands/community/rollbackbytag.html